torch/autograd/function.py

```diff
@@ -139,8 +139,11 @@ class Function(with_metaclass(FunctionMeta, _C._FunctionBase, _ContextMethodMixi
         >>>         return grad_output * result
     """
 
-    # only for backward compatibility
```

Comment:
Maybe update the comment above by removing "Each function object is meant to be used only once (in the forward pass)." as it is not relevant.
And add to the code sample:
```
# Use it by calling the apply method:
output = Exp.apply(input)
```